spec/unit/messages/users_list_message_spec.rb

```diff
@@ -84,10 +99,32 @@ module VCAP::CloudController
         expect(message).to be_valid
       end
 
+      it 'accepts partial_usernames and origins' do
```

Comment:
The `it 'accepts no usernames no origins'` seems misplaced. Should be an own context/describe.